clustering.ipynb

```diff
@@ -64,6 +64,8 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false"><span style="background-color: white; font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;">Line #3.    for key,value in obj.iteritems():
</span></pre><p>obj.iteritems() result a tuple of (key,value)</p><p>You could print out the resulted value in each iteration as one value, as it results the same : (a comma separated (key, value) in each line)</p><p>You could do:</p><p>for key_value in obj.iteritems():</p><p>&nbsp;&nbsp;print(key_value)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>